instrumentation/racecar/opentelemetry-instrumentation-racecar.gemspec

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/instrumentation/racecar/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-instrumentation-racecar'
+  spec.version     = OpenTelemetry::Instrumentation::Racecar::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'Racecar instrumentation for the OpenTelemetry framework'
+  spec.description = 'Racecar instrumentation for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.6.0'
+
+  spec.add_dependency 'activesupport'
```

Comment:
nit because it feels like quite the edge case, but i guess i have some misgivings about requiring activesupport explicitly as a dependancy. I'd almost rather just fail the compatibility check if the appropriate active support classes/modules aren't present, ala what active model serializer instrumentation does

https://github.com/open-telemetry/opentelemetry-ruby-contrib/blob/5d92b37671c59d8aff86d17e8661c3fc25da6a21/instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb#L23-L25

Reply:
yes, I was expecting feedback along these lines.

I can't make the SIG, but I will adjust based on the decision there.

Reply:
👍 I'll get this pushed later

Reply:
right, should be ready now